Makefile

```diff
@@ -1,11 +1,33 @@
 PACKAGES=$(shell go list ./...)
 OUTPUT?=build/tendermint
 
-BUILD_TAGS?='tendermint'
-LD_FLAGS = -X github.com/tendermint/tendermint/version.GitCommit=`git rev-parse --short=8 HEAD` -s -w
+BUILD_TAGS?=tendermint
+LD_FLAGS = -X github.com/tendermint/tendermint/version.GitCommit=`git rev-parse --short=8 HEAD`
 BUILD_FLAGS = -mod=readonly -ldflags "$(LD_FLAGS)"
 HTTPS_GIT := https://github.com/tendermint/tendermint.git
 DOCKER_BUF := docker run -v $(shell pwd):/workspace --workdir /workspace bufbuild/buf
+CGO_ENABLED ?= 0
+
+# handle nostrip
```

Comment:
I am confused. golang compiler does not strip info by default. Shouldn't we call this `strip`?

Also, our release binaries now will be bigger because you've removed `-s -w` flags.

Reply:
> I am confused. golang compiler does not strip info by default. Shouldn't we call this strip?

Golang does not strip by default, but we do - that's why I thought it was more approriate to call this `nostrip`. 

> Also, our release binaries now will be bigger because you've removed -s -w flags.

Yes, when you pass `nostrip` then `-s`, `-w`, and `-trimpath` are removed. Interface-wise, except for the removal of `build_c` and `install_c`, nothing really changed - we're still appending `-w -s` by default.

Reply:
Sorry, I don't understand. For us to append -s-w, we'd have to write something like

```
ifneq (,$(findstring nostrip,$(TENDERMINT_BUILD_OPTIONS)))
```

note `ifneq`, not `ifeq`

Reply:
In that case in order to append `-w -s` (and `-trimpath` to `go build` args) we would need to pass `TENDERMINT_BUILD_OPTIONS=nostrip` - and that'd be wrong. I've just preserved the current defaults (meaning: `-w -s` are on by default) and handled the `nostrip` as an option.

In other words, stripping symbols was and still is the default - my guess was that we do care about binary's size.

Reply:
Oh, I missed the fact that there's no nostrip before the comma. Sorry.